nf_core/pipeline-template/subworkflows/local/utils_nfcore_pipeline_pipeline/meta.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "pipeline_initialisation"
+description: Initialise the pipeline parameters, validate them and prepare the samplesheet
+keywords:
+  - initialisation
+  - validation
+  - parameters
+components: []
+input:
+  - version:
+      type: boolean
+      description: Display version and exit
+  - validate_params:
+      type: boolean
+      description: Boolean whether to validate parameters against the schema at runtime
+  - monochrome_logs:
+      type: boolean
+      description: Do not use coloured log outputs
+  - nextflow_cli_args:
+      type: array
+      description: List of positional nextflow CLI args
+  - outdir:
+      type: string
+      description: The output directory where the results will be saved
+  - input:
+      type: string
+      description: Path to input samplesheet
+output:
+  - samplesheet:
+      description: Channel with the samplesheet
+      structure:
+        - meta:
+          type: map
+          description: Metadata map
+        - fastq:
+          type: File
+          description: Fastq file
+          pattern: "*.fastq"
```

Comment:
the samplesheet can be csv, tsv, yamls or json

Reply:
Yeah but this is the output of the read samplesheet.
In the template it only contains the metamap and fastq files, no ?